hero.css

```diff
@@ -0,0 +1,91 @@
+@import url("./src/components/ui-lib.css");
+
+.hero {
+  font-family: sans-serif;
+  text-align: center;
+  position: relative;
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* background */
+.hero-background {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  filter: brightness(15%);
+}
+
+/* content */
+.hero-content {
+  max-width: 37.5rem;
+  z-index: 1;
+  color: var(--WHITE);
```

Comment:
better to give naming like bg-white, text-white like this rather than giving direct name of the color